app/javascript/components/lease_ups/LeaseUpApplicationsTable.js

```diff
@@ -2,26 +2,25 @@ import React from 'react'
 import ReactTable from 'react-table'
 import { trim } from 'lodash'
 
-import Dropdown from '../molecules/Dropdown'
+import StatusDropdown from '../molecules/StatusDropdown'
+import { getLeaseUpStatusClass } from '~/utils/statusUtils'
 import appPaths from '~/utils/appPaths'
 import { cellFormat } from '~/utils/reactTableUtils'
 import classNames from 'classnames'
 
-import { LEASE_UP_STATUS_OPTIONS, PAGE_SIZE, getLeaseUpStatusStyle } from './leaseUpsHelpers'
+const PAGE_SIZE = 5
```

Comment:
I would eventually like to move this to a constants file of some kind as part of a bit of refactor of how we are organizing these components. We can address that in another chore.

Reply:
I think this is actually already changed in the PR I had to page the applications that has not been implemented yet

Reply:
Ok cool, sounds like we will be handling it in that story then 👍